src/commands/environments/index.ts

```diff
@@ -28,7 +29,13 @@ export default class Environments extends Command {
     for (const env of environments) {
       // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
       // @ts-ignore
-      table.push([env.name, env.account.name, env.namespace, env.created_at, env.updated_at]);
+      table.push([
+        env.name,
+        env.account.name,
+        env.namespace,
+        localizedTimestamp(env.created_at),
+        localizedTimestamp(env.updated_at)
```

Comment:
Can you make sure to fix lint warnings. Also the ones outside of this pr. See "Unchanged files with check annotations" at the bottom.